client/archonMaker.js

```diff
@@ -226,12 +228,15 @@ const getCurvedFontSize = (length) => {
 
 const getCircularText = (text = '', diameter, kerning) => {
     return new Promise((resolve, reject) => {
```

Comment:
this probably doesnt need to be a promise either

Reply:
it does until i fix another thing

Reply:
fixed the other thing